### DISCUSSION TITLE
Building winget

### DISCUSSION THREAD

**Discussion by ajbrehm** [microsoft/winget-cli#730] [Q&A]
When I try to build the solution it reaches this point and then hangs forever (two hours so far):

15>Previous IPDB not found, fall back to full compilation.
15>All 26507 functions were compiled because no usable IPDB/IOBJ from previous compilation was found.
15>Finished generating code
15>AppInstallerCLITests.vcxproj -> C:\Users\ajbrehm\source\repos\winget-cli\src\x64\Release\AppInstallerCLITests\AppInstallerCLITests.exe
15>        1 file(s) copied.

What am I doing wrong?

**Comment by JohnMcPMS** [comment]
Any idea which project(s) are still left unfinished while it is hung?
